### ISSUE TITLE
307 Internal Redirect

### ISSUE THREAD

**Issue by pedrouid** [brave/brave-browser#2414]
Since I've migrated to the new Chromium-based version I've been getting a lot of unresolved HTTP Requests that get 307 (Internal Redirect) status codes

I get this when going through OAuth with Netlify or watch a video on SlidesLive.com or even on my own project's development website running on localhost.

I'm not sure if this is a duplicate Issue but the easiest way to reproduce would be to to login into https://app.netlify.com/ using Github

**Comment by pedrouid** [comment]
Brave Browser Version 0.57.18 Chromium: 71.0.3578.80 (Official Build) (64-bit)
Running on MacOS High Sierra 10.13.6 (17G65)

**Comment by pedrouid** [comment]
After looking at other issues and inspecting the Network tab. I get the same 307 status codes on open.spotify.com 

Duplicate of #2413 and #1391

**Comment by pedrouid** [comment]
Correction: This is not a duplicate of the two issues above!!

https://open.spotify.com was fixed by installing Widevine but https://app.netlify.com still doesnt work and still returns `307 Internal Redirect` on login

Some errors displayed:
```
Access to fetch at 'https://api.netlify.com/api/v1/user' from origin 'https://app.netlify.com' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: Redirect is not allowed for a preflight request.
```
```
Access to XMLHttpRequest at 'data:text/plain,' (redirected from 'https://api.segment.io/v1/p') from origin 'null' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.
```

I have also found this issue on other websites

**Comment by pjcarly** [comment]
I'm experiencing the same issues since upgrading.

**Comment by pjcarly** [comment]
My guess its the built in adblocking / tracking protection that is causing the issues, when I disable it, all works as it should.

**Comment by iefremov** [comment]
Should be fixed together with #2252
